CONTRIBUTING.md

```diff
@@ -12,45 +12,57 @@ Pro tip: Use the table of contents in the top-left corner to jump to specific se
-3. Make changes to the project on that branch based on your issue.
-4. [Create a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). After you had opened the pull request, there will be instructions on the [pull request template](https://github.com/hackforla/CivicTechJobs/blob/main/.github/pull_request_template.md?plain=1) for you to follow.
-5. Done! Now just wait for a team member to get back to you with feedback on your changes!
 
-> Important note 1: **We do not want to be flooded with pull requests!** Please, do **NOT** take on another issue or open a second pull request until your current pull request and linked issue are resolved by a team member.
+Option 2:
+1. [From your local repository, create a branch off of main.](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell)
+2. Make changes to the project on that branch based on your issue.
+3. [Create a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). After you had opened the pull request, there will be instructions on the [pull request template](https://github.com/hackforla/CivicTechJobs/blob/main/.github/pull_request_template.md?plain=1) for you to follow.
+3. Check that all GitHub checks passed.  This is at the bottom of the pull request.
+4. Done! Now just wait for a team member to get back to you with feedback on your changes!  (team member) To take on another issue, please ask the development lead for an appropriate additional issue to take on.
 
-> Important note 2: **Do not directly contact the team to review your pull request unless it has been over 72 hours since the pull request was opened.** As an open-source project, the team contribute to the project in their free time, which varies from week to week, day to day. We will, however, gladly answer any and all questions you may have.
+### Creating Issues
 
-## Contributing to our project (for members of our dev team)
+To create an issue as a member of our team, locate the appropriate template for your issue from the [new issue screen](https://github.com/hackforla/CivicTechJobs/issues/new/choose). Typically, you will not go wrong with [Blank Issue](https://github.com/hackforla/CivicTechJobs/issues/new?assignees=&labels=&template=blank-issue.md&title=) or the [form version](https://github.com/hackforla/CivicTechJobs/issues/new?assignees=&labels=&template=blank-issue-form.yml). 
+
+(team member) Once you are finished, [add your newly opened issue to our Project Management project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar).
```

Comment:
There's no way you could have known this, but this line isn't actually needed any longer. We have since added automation that puts issue into the board directly.